virtool/startup.py

```diff
@@ -349,6 +351,26 @@ async def init_version(app: typing.Union[dict, Application]):
     app["version"] = version
 
 
+async def init_b2c(app: Application):
+    b2c_client_id = os.environ.get("B2C_CLIENT_ID")
+    b2c_client_secret = os.environ.get("B2C_CLIENT_SECRET")
+    b2c_tenant = os.environ.get("B2C_TENANT")
+    b2c_user_flow = os.environ.get("B2C_USER_FLOW")
+    b2c_authority = os.environ.get("B2C_AUTHORITY")
```

Comment:
Why are we still using environmental variables here?

Reply:
I thought you meant that it was fine to feed in environment variables, just not overwrite them to maintain unidirectional flow. These are taken from the shell. Should I only take them as click options?